integration_tests/test_basic.py

```diff
@@ -237,18 +241,39 @@ def test_remove_from_timeslot(apps, as_admin):
 
 def test_delete_participant(apps, as_admin):
 
+    ''' Test if the researcher can delete a participant'''
+
     Experiment = apps.backend.get_model('experiments', 'Experiment')
     Location = apps.backend.get_model('experiments', 'Location')
     apps.backend.load('leader.json')
     location = Location.objects.create(name="Test Lab")
     page = as_admin
     
-    page.goto(f"{apps.backend.url}/participants/1/del/")
+    page.goto(f"{apps.backend.url}/participants/2/del/")
     page.get_by_role("button", name="Delete participant").click()
 
     cell = page.locator("td.dtr-control.sorting_1", has_text="1")
 
     assert cell.count() == 0
 
+def test_invite_participant(apps, as_admin):
 
+    ''' Testing if the researcher can invite participants and only the right participant
+    is in the list of avaible participants'''
+    
+    Experiment = apps.backend.get_model('experiments', 'Experiment')
+    Location = apps.backend.get_model('experiments', 'Location')
+    apps.backend.load('leader.json')
+    location = Location.objects.create(name="Test Lab")
+    page = as_admin
+    
+    page.goto(f"{apps.backend.url}/experiments/2/invite/")
+    assert page.locator("text=Han S. Olo").count() == 0
```

Comment:
It wasn't immediately obvious to me why this participant should be excluded (dyslexia?) so maybe add a comment for future reference.